bluefoglite/common/topology.py

```diff
@@ -14,7 +14,7 @@
 # ==============================================================================
 
```

Comment:
didn't check this file too much. Let me know if you made major modification and I can focus on that part

Reply:
I just inherit from topology in bluefog here.